mopro-ffi/src/app_config/ios.rs

```diff
@@ -115,27 +124,33 @@ pub fn build() {
         .unwrap()
         .wait()
         .unwrap();
+
+    // The iOS project expects the module map to be named "module.modulemap",
+    // but uniffi-bindgen creates "moproFFI.modulemap" by default,
+    // therefore we need to rename it.
+    rename_module_map_recursively(&bindings_out);
+
     if let Ok(info) = fs::metadata(&bindings_dest) {
         if !info.is_dir() {
             panic!("framework directory exists and is not a directory");
         }
         fs::remove_dir_all(&bindings_dest).expect("Failed to remove framework directory");
     }
+
     fs::rename(&bindings_out, &bindings_dest).expect("Failed to move framework into place");
     // Copy the mopro.swift file to the output directory
-    cleanup_tmp_local(&build_dir_path)
+    cleanup_tmp_local(build_dir_path)
 }
 
-pub fn write_bindings_swift(out_dir: &Path) {
-    if let Ok(info) = fs::metadata(out_dir) {
-        if !info.is_dir() {
-            panic!("out_dir exists and is not a directory");
+fn rename_module_map_recursively(bindings_out: &PathBuf) {
```

Comment:
Could you write a comment explaining what this function does? From context it moves a file named `moproFFI.modulemap` to `module.modulemap` in the same directory. It does this exactly once, so it doesn't fully traverse the directory structure if not needed.